### ISSUE TITLE
Can we ask one bead per heavy atom? [question]

### ISSUE THREAD

**Issue by UnixJunkie** [tbereau/auto_martini#44]
Also, is the obtained "CG molecule" still flexible if the input molecule was?

**Comment by UnixJunkie** [comment]
Or, "can we constrain the obtained CG model to have one bead per input molecule heavy atom"?

**Comment by tbereau** [comment]
Yes, you can, but you'll have to fiddle with the parameters of the model. See objective function in the JCTC paper for detail. I don't know which values you'll need though, so you'll need to look into it.
